REST/Documentation.json

```diff
@@ -1562,6 +1562,183 @@
+            "in": "path",
+            "description": "ID of the application you want to work with.",
+            "required": true,
+            "type": "integer"
+          },
+          {
+            "name": "userid:int",
+            "in": "path",
+            "description": "ID of the user.",
+            "required": true,
+            "type": "integer"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK: The request has succeeded. The information returned with the response is dependent on the method used in the request."
+          },
+          "400": {
+            "description": "Bad Request: The request could not be understood by the server due to malformed syntax. The client SHOULD NOT repeat the request without modifications."
+          },
+          "401": {
+            "description": "Unauthorized: The request requires user authentication. The client MAY repeat the request with a suitable Authorization header field. If the request already included Authorization credentials, then the 401 response indicates that authorization has been refused for those credentials."
+          },
+          "403": {
+            "description": "Forbidden: The server understood the request, but is refusing to fulfill it. Authorization will not help and the request SHOULD NOT be repeated."
+          },
+          "404": {
+            "description": "Not Found: The server has not found anything matching the Request-URI. No indication is given of whether the condition is temporary or permanent."
+          },
+          "500": {
+            "description": "Internal Server Error: The server encountered an unexpected condition which prevented it from fulfilling the request."
+          },
+          "503": {
+            "description": "Service Unavailable: The server is currently unable to handle the request due to maintenance of the server or because the service is not running."
+          }
+        },
+        "security": [
+          {
+            "ApiKey": []
+          }
+        ]
+      },
+      "put": {
+        "tags": [
+          "Users"
+        ],
+        "summary": "Set an image for the user",
+        "description": "",
+        "operationId": "PutImageByUserId",
+        "consumes": [
+          "multipart/form-data",
+          "application/x-www-form-urlencoded",
+          "multipart/form-data"
+        ],
+        "produces": [
+          "image/jpeg"
+        ],
+        "parameters": [
+          {
+            "name": "applicationid:int",
+            "in": "path",
+            "description": "ID of the application you want to work with.",
+            "required": true,
+            "type": "integer"
+          },
+          {
+            "name": "userid:int",
+            "in": "path",
+            "description": "ID of the user.",
+            "required": true,
+            "type": "integer"
+          },
+          {
+            "in": "body",
```

Comment:
According to the swagger validator, this should be formData instead of body.

I believe this can be fixed by changing BodyParameter to Parameter for the PUT action since you already override  In = ParameterIn.Form

Reply:
Fixed in [721b0ce](https://github.com/gpsgate/apidocs/pull/36/commits/721b0cef82e5b9a5e098c9636379a8d4dc520852)